src/vm/ceeload.h

```diff
@@ -3202,9 +3202,9 @@ class Module
     // This is a compressed read only copy of m_inlineTrackingMap, which is being saved to NGEN image.
     PTR_PersistentInlineTrackingMapNGen m_pPersistentInlineTrackingMapNGen;
 
-#if defined(PROFILING_SUPPORTED) && !defined(DACCESS_COMPILE)
+#if defined(PROFILING_SUPPORTED) || defined(DACCESS_COMPILE)
```

Comment:
This should be `#ifdef PROFILING_SUPPORTED_DATA` instead.

Reply:
Or `#if defined(PROFILING_SUPPORTED) || defined(PROFILING_SUPPORTED_DATA)` - it is what most of the places in VM use for some reason.

Reply:
Thanks, fixed